### PR TITLE
Update gradlew

### PR DESCRIPTION
**Pull request by oroszgy**
Fixed the gradlew file to make it actually work.

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -84,6 +84,10 @@ Please set the JAVA_HOME variable in your environment to match the
 location of your Java installation."
 fi
 
+# Create the neccessary templates for IPython
+mkdir ~/.ipython/kernels/kotlin
```
Comment (happy15) [inline gradlew]: "mkdir -p" would be better cause in my case I don't even have the kernels folder.

File: gradlew [hunk 2]
```diff
@@ -84,6 +84,10 @@ Please set the JAVA_HOME variable in your environment to match the
...
+touch ~/.ipython/kernels/kotlin/kernel.json
```
Comment (happy15) [inline gradlew]: And there is no need to touch it since the build.gradle script will create it upon writing.

**Comment by hastebrot** [comment]
I had to manually create the `~/.ipython/kernels/kotlin` directory under Ubuntu 16.04. Besides this, the Kotlin kernel works good so far, even if Linux is not officially supported right now.

This fix should definitely go into `build.gradle` instead of `gradlew`. Gradle's `wrapper` task could always overwrite this change.

**Comment by ligee** [comment]
The directory creation is properly addressed by https://github.com/ligee/kotlin-jupyter/pull/9 (merged)

**Pull request closed without merging**